recipes-bsp/u-boot/u-boot_2019.04.bbappend

```diff
@@ -32,8 +32,10 @@ TFTP_SERVER_IP ?= "127.0.0.1"
 do_configure_prepend() {
     sed -i -e 's,@SERVERIP@,${TFTP_SERVER_IP},g' ${S}/include/configs/sifive-fu540.h
 
-    mkimage -A arm -O linux -T script -C none -n "U-Boot boot script" \
-        -d ${WORKDIR}/${UBOOT_ENV}.txt ${WORKDIR}/boot.scr.uimg
+    if [ -f "${WORKDIR}/${UBOOT_ENV}.txt" ]; then
+        mkimage -A arm -O linux -T script -C none -n "U-Boot boot script" \
+            -d ${WORKDIR}/${UBOOT_ENV}.txt ${WORKDIR}/boot.scr.uimg
+    fi
 }
 
 do_deploy_append () {
```

Comment:
I'm not well known about u-boot.
But can you check my diff when do_deploy issue at ```machine=qemuriscv64``` ?
I add a little code from you work #137.

```diff
diff --git a/recipes-bsp/u-boot/u-boot_2019.04.bbappend b/recipes-bsp/u-boot/u-boot_2019.04.bbappend
index f286d4b..5d7e6a6 100644
--- a/recipes-bsp/u-boot/u-boot_2019.04.bbappend
+++ b/recipes-bsp/u-boot/u-boot_2019.04.bbappend
@@ -39,8 +39,10 @@ do_configure_prepend() {
 }
 
 do_deploy_append () {
+    if [ -f "${WORKDIR}/${UBOOT_ENV}.txt" ]; then
       install -d ${DEPLOY_DIR_IMAGE}
       install -m 755 ${WORKDIR}/boot.scr.uimg ${DEPLOY_DIR_IMAGE}/
+    fi
 }
```

Reply:
Thanks, I have updated the patch